staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -77,6 +77,12 @@ if (Test-Path "c:\k\bootstrap-config") {
   $paths += "c:\k\bootstrap-config"
 }
 
+
+if (Test-Path "c:\k\credential-provider-config.yaml") {
```

Comment:
Please also create a symbol link for this new file in https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/provisioningscripts/loggenerator.ps1